tests/01_eradiate/01_unit/scenes/surface/test_dem.py

```diff
@@ -127,21 +127,39 @@ def make_dataarray(data: pint.Quantity, coords: dict):
             },
             [(-0.089832, 0.179664), (-0.269496, 0.359328)],
         ),
+        (
+            {
+                "da": make_dataarray(
+                    data=np.zeros((10, 10)) * ureg.m,
+                    coords={
+                        "x": np.linspace(-10, 20, 10) * ureg.km,
+                        "y": np.linspace(-30, 40, 10) * ureg.km,
+                    },
+                ),
+                "geometry": "dodecahedron",
```

Comment:
I find the intent here to be not very comprehensive, maybe change the string to a value that conveys what the test is about?